src/commands/Fun Commands/FactCommand.ts

```diff
@@ -0,0 +1,19 @@
+import { RunFunction } from '../../interfaces/Command';
+import fetch from 'node-fetch';
+
+export const run: RunFunction = async (client, message) => {
+	let response: any = await fetch('https://no-api-key.com/api/v1/facts').then(
+		(res) => {
+			res.json();
+		}
+	);
+
+	await message.channel.send(
+		client.embed({ title: 'Random Fact!', description: response.fact }, message)
+	);
+};
+
+export const name: string = 'fact';
+export const category: string = 'fun';
+export const cooldown: number = 3000;
```

Comment:
exporting cooldown as 3 seconds is a bad idea as 3 is the default cooldown

Reply:
ok should I do 4 then or something

Reply:
just remove the line

Reply:
ohh, so every command has a 3000MS cooldown, and you can also specify a different cooldown?

Reply:
every command has a 3000MS cooldown by default*

Reply:
yes.

Reply:
ok thanks